### ISSUE TITLE
Update Floatplane 2FA status

### ISSUE THREAD

**Issue by Carlgo11** [2factorauth/twofactorauth#4495]
>Floatplane does have 2FA and has had it for awhile. Maybe it didn't back in April when this issue was originally created? The "authenticator app" uses the standard QR code which can be scanned with any regular 2FA app like Aegis/Authy/FreeOTP/etc. 

![image](https://user-images.githubusercontent.com/59303202/71756293-5e666480-2ed2-11ea-9bbb-6afcf719b549.png)

_Originally posted by @ukanuk in https://github.com/2factorauth/twofactorauth/pull/3803#issuecomment-570736356_

**Comment by Carlgo11** [comment]
@ukanuk could you link to any documentation page?

**Comment by ukanuk** [comment]
I'm sorry, they don't seem to have any documentation right now. For an official confirmation, a form to contact them is available at https://www.floatplane.com/support if you want, or emailing *support* at their domain floatplane.com also works.

Or there's the screenshot above which I took after logging into my account. 

Or there's this unofficial api site which talks about getting 2fa to work, though I just found them on a Google search so idk how active this project is. https://docs.floatplaneapi.com/rest-api/authentication
